### PR TITLE
Add build config for linux, add build icons

### PR DESCRIPTION
**Pull request by Krish120003**
Still need to add the 512x512 icon into the build/icons folder.

### CODE REVIEW

**Comment by seniorquico** [comment]
@Krish120003 The Linux package assets have been merged into the `master` branch.

**Review by Krish120003 — COMMENTED** [review]
Add build config for linux

File: packages/desktop-app/electron-builder.json [hunk 1]
```diff
@@ -16,5 +16,12 @@
     "artifactName": "${productName} Installer ${version}.${ext}",
     "license": "LICENSE.md",
     "oneClick": false
+  },
+  "linux": {
+    "target": ["deb", "rpm", "snap"],
+    "category": "Utility"
+  },
+  "rpm": {
+    "depends": ["libappindicator1", "libnotify-bin"]
```
Comment (seniorquico) [inline packages/desktop-app/electron-builder.json:25]: Do we know that explicitly declaring the dependencies is required? Do they not automatically include these dependencies in their template?

File: packages/desktop-app/electron-builder.json [hunk 2]
```diff
@@ -16,5 +16,12 @@
     "artifactName": "${productName} Installer ${version}.${ext}",
     "license": "LICENSE.md",
     "oneClick": false
+  },
+  "linux": {
+    "target": ["deb", "rpm", "snap"],
```
Comment (seniorquico) [inline packages/desktop-app/electron-builder.json:21]: Let's also include the AppImage target:

```suggestion
    "target": ["AppImage", "deb", "rpm", "snap"],
```

I'm not sure how popular this format is, but it was easy to download & run to app.

**Review by seniorquico — CHANGES_REQUESTED** [review]

File: packages/desktop-app/electron-builder.json (hunk 1 above)
Comment (seniorquico) [reply packages/desktop-app/electron-builder.json:25]: I created a `.rpm` build without explicitly declaring dependencies, and I tested it on CentOS 8. I was able to confirm that it does _not_ include `libappindicator1` as a dependency. The necessary dependency appears to be `libappindicator-gtk3` (e.g. Chrome & Slack depend on this package).

**Pull request closed without merging**